app/views/docs/authentication.phtml

```diff
@@ -691,76 +913,647 @@ let session = try await account.get()</code></pre>
+</p>
+
+<div class="notice">
+  <h2>Best Practice</h2>
+  <p>Only keep user sessions active as long as needed and only maintain <b>one</b> instance of the Client SDK in your app to avoid conflicting session data.</p>
+</div>
+
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large text-size-small vertical">
+    <thead>
+        <tr>
+            <th style="width: 80px"></th>
+            <th style="width: 120px">SDK</th>
+            <th>Persistence Method</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <img src="/images-ee/platforms/web.svg" alt="Logo" height="30" class="force-light sdk-logo margin-start margin-end" />
+                <img src="/images-ee/platforms/dark/web.svg" alt="Logo" height="30" class="force-dark sdk-logo margin-start margin-end" />
+            </td>
+            <td>
+                Web
+            </td>
+            <td>
+                Uses a session secure cookie and falls back to local storage when a session cookie is not available.
+            </td>
+        </tr>        
+        <tr>
+            <td>
+                <img src="/images-ee/platforms/flutter.svg" alt="Logo" height="30" class="force-light sdk-logo margin-start margin-end" />
+                <img src="/images-ee/platforms/dark/flutter.svg" alt="Logo" height="30" class="force-dark sdk-logo margin-start margin-end" />
+            </td>
+            <td>
+                Flutter
+            </td>
+            <td>
+                Uses a session cookie stored in Application Documents through the <b>path_provider</b> package.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <img src="/images-ee/platforms/apple.svg" alt="Logo" height="30" class="force-light sdk-logo margin-start margin-end" />
+                <img src="/images-ee/platforms/dark/apple.svg" alt="Logo" height="30" class="force-dark sdk-logo margin-start margin-end" />
+            </td>
+            <td>
+                Apple
+            </td>
+            <td>
+                Uses a session cookie stored in <b>UserDefaults</b>.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <img src="/images-ee/platforms/android.svg" alt="Logo" height="30" class="force-light sdk-logo margin-start margin-end" />
+                <img src="/images-ee/platforms/dark/android.svg" alt="Logo" height="30" class="force-dark sdk-logo margin-start margin-end" />
+            </td>
+            <td>
+                Android
+            </td>
+            <td>
+                Uses a session cookie stored in <b>SharedPreferences</b>.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+<h2><a href="/docs/authentication#preferences" id="preferences">User Preferences</a></h2>
```

Comment:
Not sure if preferences are relevant in an authentication context 🤔

Reply:
Yeah. I can see that. It's a part of the authentication service though :/ There's not a great place for this anywhere else

Reply:
True, better to have it somewhere than no where